seawatch_registration/views.py

```diff
@@ -54,6 +56,31 @@ def show_profile(request):
     return render(request, 'show-profile.html', {'profile': profile})
 
 
+class RequestedPositionView(LoginRequiredMixin, UserPassesTestMixin, View):
+
+    def get(self, request, *args, **kwargs):
+        return render(request, 'position.html', {'form': ProfilePositionForm(user=request.user)})
+
+    def post(self, request, *args, **kwargs):
+        form = ProfilePositionForm(request.POST, user=request.user)
+        if form.is_valid():
```

Comment:
Ich finde es besser lesbar in solchen fällen direkt das return auszugeben. so hat man eine Ebene weniger

```
if !form.is_valid():
    return render(request, 'position.html', {'form': form, 'error': 'Choose at least one position.'}) 

# continue with the happy path
 profile = form.cleaned_data['profile']
 requested_positions = form.cleaned_data['requested_positions']
…
```

Reply:
stimmt macht Sinn. Passe ich noch an